docs/userGuide/syntax/panels.md

```diff
@@ -104,6 +104,29 @@ plain text ...
 </variable>
 </include>
 
+<box>
```

Comment:
TBH i think this is too much documentation and examples.. I would rather just keep one, preferably the first because the other two are very blinding and I am old. 

In fact... I'm a bit tempted to not have documentation for this because I feel like its too small and specific feature. What do the rest of the reviewers think?

Reply:
Just one is probably good enough! But the first one is not obvious enough - maybe play with the colours a bit so it stands out more.
<img width="856" alt="Screenshot 2024-04-06 at 8 11 17 PM" src="https://github.com/MarkBind/markbind/assets/59471320/0a17ad04-b910-4032-ae75-92760b31011f">

I'm team documentation here, the use case becomes far clearer when you see how the seamless panel looks inside a colored background.

Reply:
Yep I just left one example on the user guide so that user is at least aware of this property of seamless panels. Please tell me if the colour combo is not nice 😅

Reply:
Colour looks good to me : )